readme.md

```diff
@@ -247,6 +247,7 @@ Made with Electron.
 - [Notion](https://notion.so) - All-in-one workspace for your notes, tasks, wikis, and databases.
 - [FATpick](https://www.fatpick.com) - Guitar tablature viewer that scores your performance as you play along.
 - [Taskade](https://www.taskade.com) - Realtime organization and collaboration tool for distributed teams with tasks, notes, and chat.
+- [Coloban](https://www.coloban.com) - Save hours per week on Project Management with Chats, Screenshare, Kanban, Gantt and many more.
```

Comment:
The description should be an objective description of the app, not a marketing tagline.

Reply:
Hi Sindre,
Sure, sorry - have corrected the description.